.circleci/config.yml

```diff
@@ -19,58 +20,49 @@ push_builds_only: &push_builds_only
 executors:
   amphtml-docker-small-executor:
     docker:
-      - image: cimg/base:2021.05
+      - image: cimg/base:stable
     resource_class: small
   amphtml-medium-executor:
-    machine:
-      image: ubuntu-2004:202010-01
+    docker:
+      - image: cimg/node:lts-browsers
     resource_class: medium
   amphtml-large-executor:
-    machine:
-      image: ubuntu-2004:202010-01
+    docker:
+      - image: cimg/node:lts-browsers
     resource_class: large
   amphtml-xlarge-executor:
-    machine:
-      image: ubuntu-2004:202010-01
+    docker:
+      - image: cimg/node:lts-browsers
+    resource_class: xlarge
+  validator-xlarge-executor:
```

Comment:
Similar nit here, re: consistently using / not using `docker-` in the executor name.